### ISSUE TITLE
error: relation "model" does not exist

### ISSUE THREAD

**Issue by sagic** [tjwebb/sails-permissions#172]
i started up with a new installation. updated the global npm and sails packages, followed the instruction to install sails-permission, connected a postgres db and lifted the sails.
i'm getting the error below... anybody?

(using mac)

```
info: Starting app...

info: permissions: initializing sails-permissions hook
error: Error (E_UNKNOWN) :: Encountered an unexpected error
error: relation "model" does not exist
    at Connection.parseE (/test-site/node_modules/pg/lib/connection.js:539:11)
    at Connection.parseMessage (/test-site/node_modules/pg/lib/connection.js:366:17)
    at Socket.<anonymous> (/test-site/node_modules/pg/lib/connection.js:105:22)
    at Socket.emit (events.js:107:17)
    at readableAddChunk (_stream_readable.js:163:16)
    at Socket.Readable.push (_stream_readable.js:126:10)
    at TCP.onread (net.js:538:20)

Details:  error: relation "model" does not exist

error: A hook (`sails-permissions`) failed to load!
error: Error (E_UNKNOWN) :: Encountered an unexpected error
error: relation "model" does not exist
    at Connection.parseE (/test-site/node_modules/pg/lib/connection.js:539:11)
    at Connection.parseMessage (/test-site/node_modules/pg/lib/connection.js:366:17)
    at Socket.<anonymous> (/test-site/node_modules/pg/lib/connection.js:105:22)
    at Socket.emit (events.js:107:17)
    at readableAddChunk (_stream_readable.js:163:16)
    at Socket.Readable.push (_stream_readable.js:126:10)
    at TCP.onread (net.js:538:20)

```

**Comment by ryanwilliamquinn** [comment]
what version of sails permissions/sails auth are installed?

**Comment by sagic** [comment]
sails-permissions: ^1.4.5
sails-auth: 1.3.1

i switched to sails-local for persistence and it executes smoothly... it's probably related to my postgres config. i will update after i'll figure out how to configure the db.

thanks

**Comment by GeekLad** [comment]
I'm having the same problem using sails-postgresql.  Did you have any luck getting this fixed?  I'm using sails-permissions ^1.4.5 and sails-auth: ^1.3.1

I had extended the User object to add an email_confirmed boolean field to add some functionality for sending/receiving email confirmations on new accounts, and it was causing some errors since it wasn't in the schema.  I took the lazy route and dropped all my tables, thinking that sails-permissions would recreate everything for me.  Now I can't use postgresql with my app at all :(

**Comment by GeekLad** [comment]
I ran --verbose and figured it out when I saw this line:
```Forcing Waterline to use `migrate: "safe" strategy (since this is production)```

I had my development config set to local disk db, and production set to an empty postgres instance (empty because I blew it away).  So I just had to change my development config to use the postgres instance and rebuild the tables by running a sails lift in dev mode.
